CHANGELOG.md

```diff
@@ -521,6 +520,13 @@ Released with 1.0.0-beta.37 code base.
 ### Security
 -  Ran `npm audit fix` to address vulnerabilities and update libraries (#4719) (#4728)
 
+
 ## [Unreleased]
 
-## [1.8.0]
\ No newline at end of file
+## [1.7.2]
+
+### Security
+-  `npm audit fix` to update libraries (#4860)
+
+### Fixed
+-  Fix jsonrpc payload and response types (#4743) (#4761)
```

Comment:
PRs 4743 4761 were not released under 1.7.1, https://github.com/ChainSafe/web3.js/releases/tag/v1.7.1